src/plugins/saved_objects/public/saved_object/helpers/apply_es_resp.ts

```diff
@@ -29,13 +28,13 @@ import { IndexPattern } from '../../../../data/public';
 export async function applyESResp(
   resp: EsResponse,
   savedObject: SavedObject,
-  config: SavedObjectConfig
+  config: SavedObjectConfig,
+  createSearchSource: DataPublicPluginStart['search']['createSearchSource']
```

Comment:
NIT: `ISearch['createSearchSource']`

Reply:
Tried to do this initially, but `ISearchStart` is not exported from the data plugin.

Reply:
@elastic/kibana-app-arch ^ any reason for that?